app/src/main/java/com/github/sdpteam15/polyevents/util/ZoneAdapter.kt

```diff
@@ -0,0 +1,42 @@
+package com.github.sdpteam15.polyevents.util
+
+import com.github.sdpteam15.polyevents.model.Zone
+import com.google.firebase.firestore.DocumentSnapshot
+
+/**
+ * A class for converting between zone entities in our code and
+ * documents in the Firebase database. Not unlike the DTO (Data
+ * transfer object) concept.
+ *
+ * IMPORTANT: This should be updated whenever we add, remove or update fields of UserEntity.
+ */
+class ZoneAdapter {
+    companion object {
```

Comment:
If all the methods of this class are in the companion object, can't we just declare this class as an object (i.e. a singleton) directly ? (same goes for the other adapters)

Reply:
Yes! I agree, I implemented it as a class at first because I thought I would be instantiating with the document as argument. But no longer the case good find.